test/integration/start_stop_delete_test.go

```diff
@@ -50,7 +50,6 @@ func TestStartStop(t *testing.T) {
 				"--kvm-network=default",
 				"--kvm-qemu-uri=qemu:///system",
 				"--disable-driver-mounts",
-				"--keep-context=false",
```

Comment:
We do not want to not test this option ! we still want to test this option, but make sure the test is looking at the right thing.